MapifyEditor/Export/MapExporter.cs

```diff
@@ -74,6 +73,13 @@ private static void ExportRelease(string mapName)
             {
                 EditorUtility.DisplayProgressBar("Mapify", "Creating zip file", 0);
 
+                if (File.Exists(exportFilePath))
+                {
+                    File.Delete(exportFilePath);
+                }
+
+                ZipFile.CreateFromDirectory(mapExportFolder, exportFilePath, CompressionLevel.Fastest, true);
+
```

Comment:
Was this intentional? It's a direct copy of the code below, but using the wrong delete method.

Reply:
No, that's a mistake. Fixed.